groth16_verifier/README

```diff
@@ -0,0 +1,28 @@
+# Groth16 Verifier Contract
+
+A demonstration of a Groth16 zero-knowledge proof verifier implemented as a Soroban smart contract.
+
+The proof and verification key are generated following the Circom2 [getting-started guide](https://docs.circom.io/getting-started/installation/).
```

Comment:
Could we mention what version of circom2 was used, so that someone can reproduce the inputs?

Reply:
addressed in the other PR